### ISSUE TITLE
Carousel sliding doesn't work if Windows 10 animations are turned off with new bootstrap only

### ISSUE THREAD

**Issue by FrancisGuindon** [twbs/bootstrap#30143]
- Operating system and version (Windows, macOS, Android, iOS, Win10 Mobile)
Windows 10 Version 1909
- Browser and version (Chrome, Firefox, Safari, IE, MS Edge, Opera 15+, Android Browser)
Edge w/ Chromium (Version 80.0.361.48 (Official build) beta (64-bit)), Chrome and Firefox

This StackOverflow post and solution I created earlier today summaries it well.
https://stackoverflow.com/questions/60108762/new-bootstrap-doesnt-support-carousels

Basically, if Windows 10 animations are turned off and you use bootstrap version >4.0.0, then the carousel sliding animation won't work. If you're looking for the exact setting in Windows 10, it is called "Show animations in Windows".

**Comment by ffoodd** [comment]
You don't see pictures at all, or only the sliding animation?

**Comment by FrancisGuindon** [comment]
> You don't see pictures at all, or only the sliding animation?

It is just the sliding animation that is lacking. The rest shows up fine.

**Comment by ffoodd** [comment]
Ok then, this is an accessibility feature and not a bug — you can see where it's done in [the `transition` mixin](https://github.com/twbs/bootstrap/blob/v4-dev/scss/mixins/_transition.scss).

Bootstrap uses the `prefers-reduced-motion` media query to reset transitions, and in Windows 10 I guessed this preference is exposed by the system to the browser by the setting you identified.

Maybe Windows should explain more clearly what this settings does and how it can be used on a webpage…

From Bootstrap, you still can unset this feature by setting `$enable-prefers-reduced-motion-media-query` to `false` — but it requires you compile the Sass code by yourself.

I hope it helps!

**Comment by FrancisGuindon** [comment]
I have no idea what sass is (I just know it exists), but I'm actually willing to learn by myself on how that works. I am definitely going to bed smarter tonight. Thank you!

**Comment by shadilahham** [comment]
This has been helpful. It is quite annoying that browsers pick up that setting from windows. Not sure what turning off animations in the OS has to do with turning off animations in the browser ... anyway good to know how to work around this issues for now.
I think the sass variable should be false be default. This seems like an opt-in rather than an opt-out feature, but that's just a point of view.
